### DISCUSSION TITLE
Duda respecto a función REPORTE-PEDIDOS

### DISCUSSION THREAD

**Discussion by BenjaCarrill0** [IIC2133-PUC/2024-1#51] [Tarea 0] [answered]
Hola! mientras estaba realizando los tests me di cuenta que en los archivos de outputs correctos al realizar la función REPORTE-PEDIDOS los pedidos están ordenados de forma que todos los que son de un mismo planeta están juntos como se puede ver en la siguiente imagen (es el test 1-3):   
![image](https://github.com/IIC2133-PUC/2024-1/assets/141954467/02de1357-0020-4b9a-af00-61902c9dba7a)
pero sucede que en el enunciado no se indica que sea necesario que los pedidos estén juntos, de hecho en el ejemplo dado no están así: 
![image](https://github.com/IIC2133-PUC/2024-1/assets/141954467/8be19593-cc2b-439e-b9ff-46380c48d856)
por lo que mi duda es ¿es necesario el que los pedidos estén uno "al lado" del otro según el planeta?.

Espero su respuesta.

**Comment by PabloNavarroY** [comment, marked as answer]
En el enunciado, abajo del ejemplo de output que muestras, aparece el criterio por el cual se ordenan los pedidos.

**Reply by BenjaCarrill0** [reply]
gracias, yo era el que me había equivocado y hecho mal una implementación jajajaja
